tools/updateSelectize.R

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env Rscript
+# Retrieves a particular version of selectize:
+#  https://github.com/selectize/selectize.js
+# and selectize-plugin-a11y:
+#  https://github.com/SLMNBJ/selectize-plugin-a11y
+# Then applies patches from tools/selectize-patches.
+
+# This script can be sourced from RStudio, or run with Rscript.
+
+# =============================================================================
+# Download library
+# =============================================================================
+
+## First, download the main selectize.js and css
+library(rprojroot)
+
+version <- "0.12.4"
+dest_dir <- rprojroot::find_package_root_file("inst/www/shared/selectize")
+tag <- paste0("v", version)
+dest_file <- file.path(tempdir(), paste0("selectize.js-", version, ".zip"))
+url <- sprintf("https://github.com/selectize/selectize.js/archive/%s.zip", tag)
+
+download.file(url, dest_file)
+unzipped <- tempdir()
+unzip(dest_file, exdir = unzipped)
+
+unlink(dest_dir, recursive = TRUE)
+
+dir.create(file.path(dest_dir, "js"), recursive = TRUE)
+file.copy(
+  file.path(unzipped, paste0("selectize.js-", version), "dist", "js", "selectize.min.js"),
+  file.path(dest_dir, "js"),
+  overwrite = TRUE
+)
```

Comment:
Let's make sure to copy over `selectize.js` as well

Reply:
We no longer need to inspect the difference! I have finally figured out what we've been using for slectize.min.js. It was standalone version. I have modified `tools/updateSelectize.R` script to reflect this change via 08e84e2.